docs/admin/network-plugins.md

```diff
@@ -57,6 +57,7 @@ The plugin requires a few things:
 * The standard CNI `bridge`, `lo` and `host-local` plugins are required, at minimum version 0.2.0. Kubenet will first search for them in `/opt/cni/bin`. Specify `network-plugin-dir` to supply additional search path. The first found match will take effect.
 * Kubelet must be run with the `--network-plugin=kubenet` argument to enable the plugin
 * Kubelet must also be run with the `--reconcile-cidr` argument to ensure the IP subnet assigned to the node by configuration or the controller-manager is propagated to the plugin
+* Kubelet should also be run with the `--non-masquerade-cidr=<cidr>` argumment to ensure traffic to IPs outside this range will use IP masquerade.
```

Comment:
You may not _always_ want this behavior (but you probably do most of the time).

This is only required if for example you need:
- pod->internet traffic to work (and pod's don't have publicly routable IPs)
- pod->node traffic to work (and the host network can't route to pod IPs)